lib/Pusher.php

```diff
@@ -147,6 +153,20 @@ public function __construct($auth_key, $secret, $app_id, $options = array(), $ho
             }
         }
 
+        // Set the native notification host
+        if (isset($options['notification_host'])) {
```

Comment:
you can use `if($options['notification_host'])`. In that case the empty string will be False and use the default.